.travis.yml

```diff
@@ -12,11 +12,12 @@ branches:
     - master
 
 env:
-  - ANSIBLE_INSTALL_VERSION=2.0.2.0
-  - ANSIBLE_INSTALL_VERSION=2.1.6.0
-  - ANSIBLE_INSTALL_VERSION=2.2.3.0
-  - ANSIBLE_INSTALL_VERSION=2.3.2.0
-  - ANSIBLE_INSTALL_VERSION=2.4.1.0
+  matrix:
```

Comment:
Why a matrix here?  It's only trusty with python 2.7, unless I'm missing something.